### DISCUSSION TITLE
Is there still an android version of this

### DISCUSSION THREAD

**Discussion by mdkberry** [Athou/commafeed#1070] [Q&A] [answered]
saw it [mentioned here](https://github.com/rozbb/CommaFeed-Android-Reader) but the link doesnt go to a commafeed googleplay app

**Comment by Athou** [comment, marked as answer]
No there's no Android application.
However, the web application is fully responsive and works great on mobile. You can even install it from your browser menu:
![image](https://github.com/Athou/commafeed/assets/1256795/f8199f0e-f76a-44d7-bf2a-dfb1dd8627fd)
